ifrs17/Report/ReportStorage.ipynb

```diff
@@ -132,11 +141,13 @@
                 "\n    // Constructor",
                 "\n    public ReportStorage(IWorkspace workspace, ",
                 "\n                        Systemorph.Vertex.Pivot.Builder.Interfaces.IPivotFactory reportFactory, ",
-                "\n                        IExportVariable export) {",
+                "\n                        IExportVariable export, ",
+                "\n                        int? periodicityInMonth = null) {",
```

Comment:
this seems misplaced. It should be controlled like Year and Month. 
From a report you should be able to do: 
pv.Periodicity = 3.

Reply:
It is part of the Args